packages/fxa-content-server/tests/functional/send_sms.js

```diff
@@ -192,6 +193,13 @@ const suite = {
         .then(testElementExists(selectors.SMS_SEND.HEADER));
     },
 
+    'not now': function() {
+      return this.remote
+        .then(openPage(SEND_SMS_URL, selectors.SMS_SEND.HEADER))
+        .then(click(selectors.SMS_SEND.LINK_NOT_NOW))
+        .then(testUrlInclude('https://www.mozilla.org/'));
```

Comment:
Navigating to the site 301s to the `en-US` version, it made sense to just check first part of url.